packages/uicore/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -30,14 +32,26 @@ export const Breadcrumbs: React.FC<BreadcrumbsProps> = ({ links = [], className
       className={cx(css.breadcrumbs, className)}>
       {links.map((link: Breadcrumb, index: number) => (
         <Layout.Horizontal flex={{ align: 'center-center', justifyContent: 'flex-start' }} key={index}>
-          <Link className={css.breadcrumb} to={link.url}>
-            {link.iconProps && link.iconProps.name && (
-              <Icon size={16} padding={{ right: 'xsmall' }} {...link.iconProps} />
-            )}
-            <Text intent="primary" font={{ size: 'small' }}>
-              {link.label}
-            </Text>
-          </Link>
+          {link.onClick ? (
```

Comment:
you can pass `onClick` to `<Link>` as well I think. you just need to call `event.preventDefault()` to stop the navigation.
This will avoid code duplication.